acceptance/suites/tests/030-tk-auth/010-tk-auth-setup.rb

```diff
@@ -45,7 +45,7 @@
   end
 
   step 'restart puppetserver' do
-    on(master, 'service puppetserver restart') 
+    on(master, 'service #{options[:puppetservice]} restart') 
```

Comment:
This should be `options['puppetservice']` - the keys are actually strings instead of keywords...no idea how we ended up like this